src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -553,14 +552,14 @@ private HttpTransport transportToUse(final Credential credential) {
             request.execute().getStatusCode();
         } catch (com.google.api.client.http.HttpResponseException hre) {
             if (hre.getStatusCode() == HTTP_NOT_FOUND) {
-                LOGGER.log(INFO, "OpenShift OAuth got an unexpected 404 trying out the issuer's token endpoint", hre);
+                LOGGER.log(FINE, "OpenShift OAuth got an unexpected 404 trying out the issuer's token endpoint", hre);
             }
         } catch (SSLHandshakeException ssle) {
             String formattedMessage = "OpenShift OAuth got an SSL error when accessing the issuer's token endpoint when using the SA certificate";
             LOGGER.info(formattedMessage);
             try {
                 if (jvmDefaultKeystoreTransport == null) {
-                    LOGGER.log(INFO, "jvmDefaultKeystoreTransport was not initialized: Forcing initalization");
+                    LOGGER.log(FINE, "jvmDefaultKeystoreTransport was not initialized: Forcing initalization");
```

Comment:
keep this one also as INFO, it is poor likely to happen, so if it happens, it means there is a problem, we need to see it